Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -138,6 +140,164 @@ public class PushDeploymentController : ApiController
             }
         }
 
+        [HttpPost]
+        [HttpPut]
+        public async Task<HttpResponseMessage> OnePushDeploy(
+            [FromUri] bool isAsync = false,
+            [FromUri] string author = null,
+            [FromUri] string authorEmail = null,
+            [FromUri] string deployer = OneDeploy,
+            [FromUri] string message = DefaultMessage)
+        {
+            using (_tracer.Step("OnePushDeploy"))
+            {
+                var response = Request.CreateResponse();
+                //Checks the "type" query parameter, defaults to "static" if no query parameter is provided 
+               var deployType = Request.RequestUri.ParseQueryString()["type"];
+                if (string.IsNullOrWhiteSpace(deployType))
+                {
+                    deployType = "static";
+                }
+                var website_framework = (System.Environment.GetEnvironmentVariable("website_framework".ToLower())).ToLower();
+                //Checks the "path" queary parameter, path root at /wwwroot TBD if need to change to /home
+                string path = Request.RequestUri.ParseQueryString()["path"];
```

Comment:
>path [](start = 69, length = 4)

Refer [https://github.com/shrishrirang/kudu/blob/master/Common/Constants.cs](https://github.com/shrishrirang/kudu/blob/master/Common/Constants.cs) for how to define constant strings so that they are configurable in one place